account_fiscal_year_closing/models/account_fiscalyear_closing_abstract.py

```diff
@@ -81,8 +81,27 @@ class AccountFiscalyearClosingTypeAbstract(models.AbstractModel):
         required=True,
         default="unreconciled",
     )
-    account_type_id = fields.Many2one(
-        comodel_name="account.account.type",
-        string="Account type",
+    account_type = fields.Selection(
```

Comment:
This change requires a migration, otherwise when a database is migrated the existing records will have an empty `account_type` instead of having the value corresponding to their old `account_type_id`.
You can have a look at https://oca.github.io/OpenUpgrade for better understanding what a migration is.

Since the migration is usually not trivial and it does not really affect the module's behavior, sometimes this can be done later; this will be probably decided by whoever wants to merge this module.

If this is the case, please at least write a note in the roadmap of the module.

Reply:
create test
await you feedback